cfpb/consumerfinance.gov
# Update rental-assistance-finder to version 0.3.1

- For a summary of fixes and updates, see https://github.com/cfpb/rental-assistance-finder/releases/tag/0.3.1

## How to test this PR
1. Pull down this branch and run setup scripts, etc
2. Check the [Rental Assistance Finder](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/) for the updates outlined at the above url.


## Checklist

### Front-end testing

#### Browser testing

Visually tested in the following supported browsers:

#### Accessibility

#### Other
